x-pack/legacy/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -397,4 +397,97 @@ describe('Task Runner', () => {
       }
     `);
   });
+
+  test('recovers gracefully when the Alert Task Runner throws an exception when fetching the ApiKey', async () => {
```

Comment:
the description on this seems wrong - and in fact there is a test for "throws an exception when decrypting attributes" below, which doesn't appear to be testing decrypting encrypted attributes, but this one does.